contracts/ProcedureTable.sol

```diff
@@ -450,7 +791,7 @@ library ProcedureTable {
 
         // The capabilities are parsed here. We neeed to pass in the Procedure
         // struct as solidity can't return complex data structures.
-        _parseCaps(p,caps);
+        // _parseCaps(p,caps);
```

Comment:
Is this ommited?

Reply:
For the same reason as getProcedureByKey. It's inefficient and does not form a proper abstraction. We use the location values as defined in the spec, which is incompatible with this.

Reply:
Ok. Let's remove this if it's not needed.